### PR TITLE
added "or other spatial coordinates" as per #33

### PR DESCRIPTION
**Pull request by ChrisBarker-NOAA**
Makes it clear that other projections are allowed.

We should still explicitly call out to CF for the CRS information.

Sorry about the whitespace changes -- my editor removes trailing whitespace -- but it's probably for the better after this :-)

### CODE REVIEW

File: src/index.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
+This page describes a proposal for storing unstructured (or flexible mesh) model data in the Unidata Network Common Data Form (NetCDF) file. Our focus is on data for environmental applications and hence we start from the [Climate & Forecast (CF) Metadata Conventions](http://cf-convention.github.io/). The CF Conventions have been the standard in climate research for many years, and are  being adopted by others as the metadata standard (e.g. [NASA](https://earthdata.nasa.gov/data/standards-and-references/metadata-standards),[Open Geospatial Consortium](http://www.opengeospatial.org/standards/netcdf)). The CF conventions allow you to provide the geospatial and temporal coordinates for scientific data, but currently assumes that the horizontal topology may be inferred from the i,j indices of structured grids.  This proposal adds conventions for specifying the topology for unstructured (e.g. triangular) grids.
```
[the remark this answers is not in the document]
Comment (ChrisBarker-NOAA) [reply src/index.md]: well, that's all one paragraph, and once rendered, is OK like that.

And that's how it was 00 I just edited it a bit -- but yeah, really hard to view the diff, so some newlines are in order.

in fact, i can't actually see any difference here at all!

I'll clean it up a bit...

**Comment by ChrisBarker-NOAA** [comment]
OK, did some more whitespace changes -- which will make it easier to review later, but even harder now.

But is is a small change, from "other coordinates" to "other spatial coordinates" in a few places.

So I'm going to merge and be done with it.

**Comment by ocefpaf** [comment]
> punctuation that enforces ```.\n``` (``<period><newline>``)

I wrote a book with a friend on GitHub recently and I advocate break lines for ``<periods>``, ``<commas>``, etc. That made our lives so much easier when reviewing tons of text.

**Comment by hrajagers** [comment]
I agree that that is a good convention. We apply the same conventions for
our LaTEX manuals stored in svn. It's easier to figure out in what the
differences between revisions are.

Bert

**Comment by ChrisBarker-NOAA** [comment]
This is a big document one on page. Maybe it would be good to break it up into multiple pages...

But in the meantime, if someone wants to do a single PR with not other changes than adding newlines -- I'd be all for that.

**Pull request merged**